packages/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -301,6 +301,49 @@ describe('<lion-input-datepicker>', () => {
       await elObj.closeCalendar();
       expect(elObj.invokerEl.getAttribute('aria-expanded')).to.equal('false');
     });
+
+    it('is accessible when closed', async () => {
+      const el = await fixture(html`
+        <lion-input-datepicker></lion-input-datepicker>
+      `);
+      const elObj = new DatepickerInputObject(el);
+
+      await expect(elObj.invokerEl).to.be.accessible();
```

Comment:
Also check the full element, not only the invoker?

Reply:
I was hinting on the `el` to be accessible, since it would make sense to have the an `aria-describeby` on the input to the invoker. 

The calendar is hidden, so not sure what to test about that.